docs/website/docs/getting-started/index.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started Guide
+
+## Setup
+Setup your local environment by installing IREE's prebuilt binaries. See the comprehensive
+installation [guide](../bindings/python.md) for installation of individual components, the default installation is below.
+
+```
+python -m pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tf \
+  iree-tools-tflite \
+  iree-tools-xla
+```
+
+## Supported frameworks
+
+See end-to-end examples of how to execute a variety models on IREE. This covers
+the import, compilation, and execution of the provided model.
+
+* [TensorFlow](./tensorflow.md)
+* [TensorFlow Lite](./tflite.md)
+* [JAX](./jax.md)
+
+Importing from PyTorch and other frameworks is planned - stay tuned!
+
+## Samples
+
+Check out the samples in IREE's
+[colab/ directory](https://github.com/google/iree/tree/main/colab) and the
+[iree-samples repository](https://github.com/google/iree-samples) for examples
+and workflow comparisons across frameworks.
+
+## Import
+
+Importing models takes known file types and imports into a form that the core IREE
+compiler is able to ingest. This import process is specific to each frontend and typically
+runs a couple of stages:
+
+* Load the source format
+* Legalize operations specific for specific frontend to a known IR
+* Validate only legal operations remain
+* Write the remaining IR to an IREE compatible format
+
+This fully legalized form can then be compiled without depending on the 
+original model format.
+
+## Compilation
+
+During compilation we load an MLIR file and compile for a set of specified backends
+(CPU, GPU, etc).  Each of these backends creates custom native code to execute on the
+target device.  Once compiled, the resulting bytecode is exported to an IREE bytecode
+file that can be executed on the specified devices.
+
+## Execution
+
+The final stage is executing the now compiled module. This involves selecting what
+compute devices should be used, loading the module, and executing the module with the
+intended inputs. For testing IREE includes a Python API however on smaller devices you
+will want to use the [C API](../deployment-configurations/index.md).
```

Comment:
let's reword this - the choice of binding (of which the C API is one) and deployment configuration involves more than just "Python" vs "small device"

```suggestion
## Execution

The final stage is executing the now compiled module. This involves selecting what
compute devices should be used, loading the module, and executing the module with the
intended inputs. IREE includes a runtime Python API for testing, however for more
complex scenarios you will want to review IREE's supported
[deployment configurations](../deployment-configurations/index.md) and
[bindings](../bindings/index.md).
```